02-atmos/Dockerfile

```diff
@@ -0,0 +1,82 @@
+ARG VERSION=0.141.6
+ARG OS=alpine
+ARG CLI_NAME=atmos
+
+FROM cloudposse/geodesic:$VERSION-$OS as cli
+
+RUN apk add -u go variant2@cloudposse
+
+# Configure Go
+ENV GOROOT /usr/lib/go
+ENV GOPATH /go
+ENV PATH /go/bin:$PATH
+
+# Build a minimal variant binary in order to download all the required libraries and save them in a Docker layer cache
+COPY cli/build-cache /tmp
+WORKDIR /tmp/build-cache
+RUN variant2 export binary $PWD variant-echo
+
+# Build the CLI
+WORKDIR /usr/cli
+COPY cli/ .
```

Comment:
I feel like we should jsut cut all this out. It makes it all feel more complicated to get started with atmos when all we need to do is `apt-get install atmos`

Reply:
@osterman do you mean pair down the Dockerfile to be simpler or cut out using Geodesic in general for this tutorial? I believe you mean the former, but just want to be double confirm. I will slim it down regardless.

Reply:
Ah you likely mean not needing to build the atmos CLI itself as part of this image I think. I'll give that a shot.

Reply:
Update to a slimmer Dockerfile and removed `cli/`. All still working as expected.